css/auth.css

```
@@ -132,14 +141,30 @@ form {
   align-items: center;
 }
 
-.signup_link {
+/* 공통 링크 스타일 */
+.auth_link {
   display: flex;
   justify-content: center;
   font-size: 14px;
   font-weight: 500;
   color: var(--gray-800);
 }
 
-.signup_link a {
+.auth_link a {
   color: var(--blue);
 }
+
+@media (min-width: 768px) and (max-width: 1199px) {
+  .auth_container {
+    padding: 0 16px;
+    width: 100%;
+  }
+}
+
+@media (min-width: 375px) and (max-width: 767px) {
+  .auth_container {
+    padding: 0 16px;
+    width: 100%;
+    max-width: 400px;
+  }
+}
```

❗️ 수정요청
미디어쿼리 조건문을 아래처럼 작성하셔서 코드 중복을 줄이시는 것을 추천드려요.

```suggestion
@media (max-width: 1199px) {
  .auth_container {
    padding: 0 16px;
    width: 100%;
  }
}

@media (min-width: 375px) and (max-width: 767px) {
  .auth_container {
    max-width: 400px;
  }
}
```